setup.py

```diff
@@ -25,6 +25,6 @@
       license='3-Clause BSD',
       packages=['splot'],
       include_package_data=True,
-      install_requires=['numpy', 'scipy', 'libpysal', 'mapclassify', 'palettable',
-                        'esda', 'pysal',],
+      install_requires=['numpy', 'libpysal', 'mapclassify', 'palettable',
+                        'esda', 'pysal', 'matplotlib', 'seaborn',],
```

Comment:
seaborn [requires](https://seaborn.pydata.org/installing.html) matplotlib , so listing matplotlib here maybe redundant.

Reply:
Ideally I would like to cut ties to `seaborn` to keep dependencies as low as possible. So keeping them both might be an easy fix later on, when `seaborn` is removed? Otherwise I am happy to change that. What do you think?

Reply:
That makes sense. So this is good to go as is from my perspective.